src/main/java/ch/njol/skript/expressions/ExprTimeSince.java

```diff
@@ -1,61 +1,57 @@
-/**
- *   This file is part of Skript.
- *
- *  Skript is free software: you can redistribute it and/or modify
- *  it under the terms of the GNU General Public License as published by
- *  the Free Software Foundation, either version 3 of the License, or
- *  (at your option) any later version.
- *
- *  Skript is distributed in the hope that it will be useful,
- *  but WITHOUT ANY WARRANTY; without even the implied warranty of
- *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- *  GNU General Public License for more details.
- *
- *  You should have received a copy of the GNU General Public License
- *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
- *
- * Copyright Peter Güttinger, SkriptLang team and contributors
- */
 package ch.njol.skript.expressions;
 
-import org.bukkit.event.Event;
-import org.jetbrains.annotations.Nullable;
-
 import ch.njol.skript.Skript;
 import ch.njol.skript.doc.Description;
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
 import ch.njol.skript.doc.Since;
 import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.skript.util.Date;
 import ch.njol.skript.util.Timespan;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
 
-@Name("Time Since")
-@Description("The time that has passed since a date. If the given date is in the future, a value will not be returned.")
-@Examples("send \"%time since 5 minecraft days ago% has passed since 5 minecraft days ago!\" to player")
-@Since("2.5")
+@Name("Time Since/Until")
+@Description({
+	"The time since a date has passed or the time until a date will pass. will return 0 seconds.",
+	"This expression will return 0 seconds if the time since or time until would be negative, eg. if one tries to get the time since a future date."
```

Comment:
```suggestion
	"This expression will return 0 seconds if the time since or time until would be negative, e.g. if one tries to get the time since a future date."
```